cmd/config_test.go

```diff
@@ -54,6 +55,9 @@ func TestTLSConfigLoad(t *testing.T) {
 	cert := "testdata/cert.pem"
 	key := "testdata/key.pem"
 	caCert := "testdata/minica.pem"
+	// Instantiate a nil Registerer to satisfy Load(scope)
+	var scope prometheus.Registerer
```

Comment:
Rather than doing this, just pass a [`metrics.NoopRegisterer`](https://github.com/letsencrypt/boulder/blob/3ddca2d1b85cddcd6b3490b0f640078fc5b1b58f/metrics/scope.go#L12) on line 93, which we made for exactly this purpose.

Same in the other two test files below.